yt_dlp/extractor/beatbump.py

```diff
@@ -0,0 +1,112 @@
+import re
+
+from yt_dlp.utils import ExtractorError
+from .common import InfoExtractor
+
+
+class BeatBumpIE(InfoExtractor):
```

Comment:
Split this into 4 extractors, for each branch

Reply:
Done in 13c7367e5f54cbbf91cac022e39c7933d37198c2